packages/autocomplete-core/src/onKeyDown.ts

```diff
@@ -32,7 +32,7 @@ export function onKeyDown<TItem>({
     // Arrow down.
     event.preventDefault();
 
-    store.send(event.key, { shiftKey: event.shiftKey });
```

Comment:
was this never used?

Reply:
Nope. I initially supported <kbd>Shift</kbd><kbd>↓</kbd> to jump items 5 by 5 but then decided that it was gadget. I forgot to remove this parameter.